File: Fibonacci.java
import java.util.Scanner;

public class Fibonacci {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Ingrese el número de términos: ");
        int n = sc.nextInt();

        int a = 0, b = 1;
        System.out.print("Serie Fibonacci: ");
        for (int i = 0; i < n; i++) {
            System.out.print(a + " ");
            int next = a + b;
            a = b;
            b = next;
        }
        System.out.println();
        sc.close();
    }
}

File: Fibonacci.c
#include <stdio.h>

int main() {
    int n, a = 0, b = 1, next, i;
    printf("Ingrese el numero de terminos: ");
    scanf("%d", &n);

    printf("Serie Fibonacci: ");
    for (i = 0; i < n; i++) {
        printf("%d ", a);
        next = a + b;
        a = b;
        b = next;
    }
    printf("\n");
    return 0;
}